### PR TITLE
Portfolio milestone 3: Created contact section

### PR DESCRIPTION
**Pull request by andres-condezo**
## Portfolio milestone 3: Created contact section
In this milestone we:

Added contact form section in HTML file.
Added styles for new sections.
Added images assets.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -213,10 +213,37 @@ <h2 class="h2-span">LET’S CONNECT</h2>
...
+					</li>
+					<li class="input">
+						<input type="email" placeholder="email" name="email" required>
+					</li>
+					<li class="input">
+						<textarea name="message" placeholder="Write your message here" rows="8" maxlength="500" required></textarea>
+					</li>
+					<li class="input">
+						<button type="submit">Get in Touch</button>
+					</li>
```
Comment (Gambit142) [inline index.html]: Great work implementing most of the requirements for this project 💪 👍. You just have one more project requirement to fulfill.

- [x] Kindly implement the button interaction of `hover`, `active`, and `focus` for this button. You can find the color to use for the button interaction on this part of your Figma template on Figma's website;

image

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @andres-condezo 👋, 

Good job so far!

## Highlight
- Good pull request title 🟢 
- Good pull request summary 🟢
- Wonderful UI design 🟢
- Implemented all form validations 🟢
- Form submits to formspree 🟢

However, there is one issue that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (andres-condezo) [reply index.html]: HI @Gambit142, I will implement the changes as soon as possible.

**Review by andersonlebon — APPROVED** [review]
Hi @andres-condezo 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub flow :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 
- Your design is looking good Kudo :clap: :heavy_check_mark: 
## Optional suggestions
- [x] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**